debian/landscape-client.postinst

```diff
@@ -106,6 +106,15 @@ END
         USER_UPDATE_FLAG_FILE="$DATA_PATH/user-update-flag"
         install --owner=landscape /dev/null $USER_UPDATE_FLAG_FILE
         echo "This file indicates that the Landscape client needs to send updated user information to the server." >> $USER_UPDATE_FLAG_FILE
+
+        # To work around bug #1735100 we rewrite file-local landscape sources
+        # with the trusted flag, as they have no release file, thus are
+        # unsigned repositories.
+        LANDSCAPE_INTERNAL_SOURCES=/etc/apt/sources.list.d/_landscape-internal-facade.list
+        if grep -q -e "^deb file:" $LANDSCAPE_INTERNAL_SOURCES; then
+            cp $LANDSCAPE_INTERNAL_SOURCES $LANDSCAPE_INTERNAL_SOURCES.orig
```

Comment:
It feels strange to leave orig this around even after we've successfully completed the package profile install. We probably should remove it when we LANDSACPE_INTERNAL_SOURCES (on success)